frontend/src/state/actions/studentActions.ts

```diff
@@ -22,7 +43,7 @@ export const setStudentAcademicYearAction = createAction(
 
 export const setStudentGraduationYearAction = createAction(
   "student/SET_GRADUATION_YEAR",
-  (graduationYear: number) => ({
+  (graduationYear: number | null) => ({
```

Comment:
Just curious, why is graduation allowed to be null?

Reply:
oh i think because right now IUserData's graduationYear can be null.  I think that's probably cuz when you first initialize a student, you have no graduation year. It kinda doesn't make sense if graduationyear is 0

Reply:
What do you have when you initialize a student?